templates/index.html

```diff
@@ -122,7 +119,7 @@ <h3 class="p-heading-icon__title">Vanilla Design</h3>
         </div>
         <p>Get our Sketch library of common design components.</p>
         <p style="margin-bottom: 0px;"><a onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage', 'eventAction' : 'Downloads', 'eventLabel' : 'Download Vanilla Design UI Kit', 'eventValue' : undefined });" class="p-button" href="https://assets.ubuntu.com/latest-redirects/vanilla-framework.sketch">Download the Sketch UI Kit</a></p>
-        <small>Filesize - 2.4 MB</small>
+        <small><i class="p-icon--warning"></i> The Sketch UI Kit is no longer maintained. Current version is compatible with Vanilla v.2.25 from March 2021.</small>
```

Comment:
Turned out quite long, let's shorten this up a bit:

```suggestion
        <small><i class="p-icon--warning"></i> The Sketch UI Kit is no longer maintained. Current version is compatible with Vanilla v2.25.</small>
```